README.md

```diff
@@ -1 +1,62 @@
-# Airflow API Client for Go
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+-->
+Airflow Go client
+=================
+
+Go Airflow OpenAPI client generated from [openapi spec](https://github.com/apache/airflow/tree/master/clients).
```

Comment:
```suggestion
Go Airflow OpenAPI client generated from [openapi spec](https://github.com/apache/airflow/blob/master/airflow/api_connexion/openapi/v1.yaml).
```
Current link results in 404

Reply:
@turbaszek I updated the link to point to `https://github.com/apache/airflow/blob/master/clients/gen/go.sh`, which will be a valid link once https://github.com/apache/airflow/pull/9502 gets merged. I think linking to the actual automation script might be better here since there is no guarantee that future of the client will be generated using the same v1.yaml spec. It also provides more context to users on how this client is actually generated so it's easier for them to contribute change back to the airflow repo.